prow/jobs/console/compass/compass.yaml

```diff
@@ -48,3 +48,10 @@ postsubmits:
       labels:
         <<: *job_labels_template
         preset-build-console-master: "true"
+    - name: post-rel-kyma-compass
```

Comment:
Please remove it for now, as in Kyma we didn't figure out how to release Kyma components, which will be in separate repositories.